helper/subproc/subproc.go

```diff
@@ -45,12 +46,15 @@ func Print(format string, args ...any) {
 //
 // r should be a buffer containing output (typically combined stdin + stdout)
 // f should be an HCLogger Print method (e.g. log.Debug)
-func Log(r io.Reader, f func(msg string, args ...any)) {
+func Log(r io.Reader, f func(msg string, args ...any)) string {
 	scanner := bufio.NewScanner(r)
+	lines := ""
 	for scanner.Scan() {
 		line := scanner.Text()
+		lines += line + "\n"
 		f("sub-process", "OUTPUT", line)
 	}
+	return strings.TrimSuffix(lines, "\n")
```

Comment:
So I should skip appending the `\n` ?

Reply:
Oh shoot, you know what, I'm totally wrong because I was looking at the docs for the methods and not the type:

> the default split function breaks the input into lines with line termination stripped.

So what you're doing here is exactly right. Probably worth swapping out `return strings.TrimSuffix(lines, "\n")` with `return strings.TrimSpace(lines)` and then you're good-to-go.